modules/storage-persistent-storage-azure-file-pod.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage/persistent-storage-azure-file.adoc
+
+[id="create-azure-file-pod_{context}"]
+= Mount the Azure File in a Pod
+
+
+
+.Prerequisites
+
+* A PersistentVolume exists that references the Azure File share.
+* This PersistentVolume has been mapped to a PersistentVolumeClaim.
+
+.Procedure
+
+* Create a Pod that mounts the existing PersistentVolumeClaim.
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod-name <1>
+spec:
+  containers:
+    ...
+    volumeMounts:
+    - mountPath: "/data" <2>
+      name: azure-file-share
+  volumes:
+    - name: azure-file-share
+      persistentVolumeClaim:
+        claimName: claim1 <4>
+----
+<1> The name of the Pod.
+<2> The path to mount the Azure File share inside the Pod.
+<3> The name of the PersistentVolumeClaim that has been previously created.
```

Comment:
s/PersistentVolumeClaim that has been previously created./existing PersistentVolumeClaim.